slack-java-client/src/test/java/com/hubspot/slack/client/SlackClientModuleTest.java

```diff
@@ -0,0 +1,14 @@
+package com.hubspot.slack.client;
+
+import org.junit.Test;
+
+import com.google.inject.Guice;
+import com.google.inject.Stage;
+
+public class SlackClientModuleTest {
+
```

Comment:
```suggestion
private static class PickyGuiceModule extends AbstractModule {
    @Override
    protected void configure() {
      binder().requireAtInjectOnConstructors();
      install(new SlackClientModule());
    }
  }
```
Can we create the guice module in this test class directly rather than having a new class? It's easier to read.

Reply:
Yep, done